### PR TITLE
Update ppp 2

### PR DESCRIPTION
**Pull request by zatkiller**
- Fix some grammar errors in PPP
- Update Model Diagram

### CODE REVIEW

**Review by tashawan23 — APPROVED** [review]
Looks good

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -142,24 +142,24 @@ The above mentioned Parser classes all inherit the `#parse` method from the Pars
...
  A ` DoneTaskCommandParser` object is created and it calls `DoneTaskCommandParser#parse` on the arguments
 
 Step 2. `DoneTaskCommandParser#parse` method will check on the validity of the arguments for a `DoneTaskCommand`. If it
-is valid,  it will call the create a new `DoneTaskCommand` by calling the constructor.
+is valid, it will create a new `DoneTaskCommand` instance by calling the constructor of `DoneTaskCommand`.
 
 Step 3. The `DoneTaskCommand#execute` is then called by the `LogicManager`. The task with the same `Index` is retrieved and
-a copy of the task is created with the same attribute values. However. the `TaskStatus` value is updated to be **completed**'
-in the `Model`.
+a copy of the task is created with the same attribute values. However. the `TaskStatus` value is updated to be `completed`. The task copy
```
Comment (rachelljt) [inline docs/DeveloperGuide.md]: I think you should replace the full stop after "However" to a comma instead

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -142,24 +142,24 @@ The above mentioned Parser classes all inherit the `#parse` method from the Pars
...
 Below is the usage scenario and how the mark the task as completed mechanism behaves.
 
 Assumptions:
 1. User has already launched the app
-2. HEY MATEz application has an existing task with **uncompleted** status
+2. HEY MATEz application has an existing task whose status is `uncompleted`
 
-Step 1. User executes the `done 1` command to mark the task with index number 1 in the task list of HEY MATEz to be completed. 
+Step 1. User executes the `done 1` command to mark the task with index number 1 in the task list of HEY MATEz to be marked as `completed`. 
  A ` DoneTaskCommandParser` object is created and it calls `DoneTaskCommandParser#parse` on the arguments
```
Comment (rachelljt) [inline docs/DeveloperGuide.md]: I think you forgot a full stop at the end of this sentence

**Review by rachelljt — APPROVED** [review]
Looks good to me, just need to change the punctuations

**Pull request merged**